### DISCUSSION TITLE
Multiple tags filtering?

### DISCUSSION THREAD

**Discussion by Geobert** [foambubble/foam#1086] [Q&A]
Hi,

I've asked on Discord but only got one answer and no solution so I'm opening this discussion here :)

Is there a way to filter notes per multiple tags? 

Let's say I want notes that have both `tag1` and `tag2`. I can't see anyway to do it with the tag explorer. The graph can give the information, but in a big graph, I think it will be hard to follow.

So am I missing something or is this question becoming a feature request? :)

Regards,

**Comment by riccardoferretti** [comment]
Right now the tag controls are additive, so if you select multiple tags it will show notes that have any of the selected tags. 
There is no way atm to do it as an "and" operation.
Technically is pretty trivial for the specific task, but doing something like in a comprehensive matter might require a bit more thinking upfront. 
Happy to support if someone wants to take a stab at it

**Comment by Lloyd-Jackman-UKPL** [comment]
This is one feature of the Obsidian graphing feature that (currently) sets it apart from Foam as the ability to have greater clarity in the graph view can be of great help in establishing where they ought to be connections.
